### PR TITLE
Added genomicconsensus recipe

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/genomicconsensus/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+$PYTHON setup.py clean
```
Comment (bgruening) [inline recipes/genomicconsensus/build.sh]: Is this step really needed?

File: recipes/genomicconsensus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "genomicconsensus" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/PacificBiosciences/GenomicConsensus/archive/smrtanalysis-{{ version }}.tar.gz
```
Comment (bgruening) [inline recipes/genomicconsensus/meta.yaml]: Can you please add a md5 checksum?

Comment (mictadlo) [inline recipes/genomicconsensus/meta.yaml]: Does github it hide it anywhere or do I have to execute md5 command?

File: recipes/genomicconsensus/build.sh (hunk 1 above)
Comment (bgruening) [inline recipes/genomicconsensus/build.sh]: Execute md5sum is what I do.

**Comment by bgruening** [comment]
Thanks @mictadlo!

**Comment by mictadlo** [comment]
Thank you @bgruening @kyleabeauchamp and @druvus

**Pull request merged**